command/shell_test.go

```diff
@@ -55,6 +55,18 @@ func (s *shellExecuteCommandSuite) TestWorksWithEmptyShell() {
 	s.NoError(cmd.Execute(s.ctx, s.comm, s.logger, s.conf))
 }
 
+
+func (s *shellExecuteCommandSuite) TestErrorToIgnoreAndRedirectToStdOut() {
+	cmd := &subprocessExec{}
+	s.NoError(cmd.ParseParams(map[string]interface{}{}))
+
+	cmd.Silent = true
+	s.NoError(cmd.ParseParams(map[string]interface{}{}))
+
+	cmd.RedirectStandardErrorToOutput = true
+	s.Error(cmd.ParseParams(map[string]interface{}{}))
+}
```

Comment:
I'm not sure that I understand why this assertion is the right one. It seems somewhat brittle. Shouldn't we assert that IgnoreStandard* is false, then run parse prams with Silent = true, and then check their value...